### ISSUE TITLE
Can not search in older releases

### ISSUE THREAD

**Issue by flip111** [commercialhaskell/stackage-content#187]
Reproduce

* Go to https://www.stackage.org/
* Click on [LTS 14.27 for ghc-8.6.5](https://www.stackage.org/lts-14.27)
* In search box type `exceptions` and click Search

Note the following:
* URL is set to: `https://www.stackage.org/lts-14.27/hoogle?q=exceptions`
* On the page there is the text `Within [LTS Haskell 22.11 (ghc-9.6.4)](https://www.stackage.org/lts-22.11)`

Continue
* Click on [package exceptions](https://www.stackage.org/lts-22.11/package/exceptions)

Note the following:
Getting package information for exceptions in LTS-22.11

**Comment by alaendle** [comment]
Yeah, a little bit misleading - but I think this is a duplicate of https://github.com/fpco/stackage-server/issues/305.

**Comment by flip111** [comment]
You are right it's a duplicate

**Comment by chreekat** [comment]
Let's call this "not a duplicate" because searching in older snapshots would be rad. I'll move the issue to stackage-server, though, since that's where it's implemented.
